### PR TITLE
internal/website: add order tutorial

### PR DESCRIPTION
**Pull request by jba**
First cut at a tutorial for the order sample.

### CODE REVIEW

File: internal/website/content/tutorials/order.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: "Order"
+date: 2019-07-09T10:22:39-04:00
+draft: true
+showInSidenav: false  # only for sections (any level)
```
Comment (eliben) [inline internal/website/content/tutorials/order.md]: Why these? Seems like it should be similar to guestbook

File: internal/website/content/tutorials/order.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
+title: "Order"
```
Comment (eliben) [inline internal/website/content/tutorials/order.md]: I'd give it a more descriptive name, like "Order Processor"

File: internal/website/content/tutorials/order.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Enter an email address (it need not be real) and select any image file from your
+computer. Then click Submit.
+
+Now visit the order list page by returning to the home page and clicking "List
+Conversions". It make take a few seconds to process the order (thanks to an
+artificial delay in the processor), so refresh the page until you see your order
+in the list.
+
+Then click on the output image link to see the converted image in your browser.
+
```
Comment (eliben) [inline internal/website/content/tutorials/order.md]: Lots of extra whitespace

File: internal/website/content/tutorials/order.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+Visit the home page in your browser and click "Convert an Image".
+
+Enter an email address (it need not be real) and select any image file from your
+computer. Then click Submit.
+
+Now visit the order list page by returning to the home page and clicking "List
+Conversions". It make take a few seconds to process the order (thanks to an
+artificial delay in the processor), so refresh the page until you see your order
+in the list.
```
Comment (eliben) [inline internal/website/content/tutorials/order.md]: Should this also say something about the changes needed to run on real cloud?

File: internal/website/content/tutorials/order.md (hunk 1 above)
Comment (jba) [reply internal/website/content/tutorials/order.md]: I don't know what they mean; `hugo new` inserted them, I think. Removed.

File: internal/website/content/tutorials/order.md (hunk 2 above)
Comment (jba) [reply internal/website/content/tutorials/order.md]: done

File: internal/website/content/tutorials/order.md (hunk 4 above)
Comment (jba) [reply internal/website/content/tutorials/order.md]: Said a little.

**Comment by jba** [comment]
PTAL

File: internal/website/content/tutorials/order.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+Platform, Amazone AWS or Microsoft Azure, you will have to provision
+some resources:
+
+- A storage bucket, to hold the image files.
+- A Pub/Sub topic and subscription, for requests from the frontend to the
+  processor.
+- A document store collection (Google Firestore, Amazon DynamoDB or Document DB,
+  or Microsoft Cosmos) to store order metadata.
+
+Then launch the `order` program with flags that provide the URLs to your
```
Comment (vangent) [inline internal/website/content/tutorials/order.md]: Please add links somewhere in here to the respective "how to" pages (e.g., https://gocloud.dev/howto/blob/ for blob), so the user can see how to create the URLs.

Comment (jba) [reply internal/website/content/tutorials/order.md]: Done.

File: internal/website/content/tutorials/order.md (hunk 3 above)
Comment (jba) [reply internal/website/content/tutorials/order.md]: Couldn't find it.

**Comment by jba** [comment]
> am wondering: shouldn't this be pointing to the source code with some kind of guidelines on how it is structure to make this into a tutorial for Go CDK and its usage? Otherwise it doesn't really "tutor", just show off for the CDK can handle.

@eliben @vangent

**Pull request merged**